### PR TITLE
Lots of stubbing out 

### PR DESCRIPTION
**Pull request by quinnelton**
Bummed that the link to the spreadsheet doesn't work, will have to find a different way to do that. 

I kept the names Skin Pack, World Template, and Resource Pack capitalized, let me know if that's against the style guide.

Labeled the page BETA.

Added some //TODOs :)

### CODE REVIEW

**Comment by mattwojo** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[minecraft-marketplace/submit-content.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/submit-content.md) | :warning:Warning | | [Details](#user-content-minecraft-marketplace%2Fsubmit-content.md)
[minecraft-marketplace/create/base-content-requirements.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/create/base-content-requirements.md) | :warning:Warning | | [Details](#user-content-minecraft-marketplace%2Fcreate%2Fbase-content-requirements.md)
[minecraft-marketplace/TOC.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/TOC.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/create-content.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/create-content.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/create/content-requirements-change-log.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/create/content-requirements-change-log.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/create/package-resource-pack.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/create/package-resource-pack.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/create/package-skin-pack.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/create/package-skin-pack.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/create/package-world-template.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/create/package-world-template.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/files/Required_Content_Info.xlsx](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/files/Required_Content_Info.xlsx) | :white_check_mark:Succeeded | |
[minecraft-marketplace/files/skinTemplate.png](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/files/skinTemplate.png) | :white_check_mark:Succeeded | |
[minecraft-marketplace/get-started.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/get-started.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/index.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/index.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/submit/content-ingestion-process.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/submit/content-ingestion-process.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/submit/marketing-assets.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/submit/marketing-assets.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/test-content.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/test-content.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/test/JIRA.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/test/JIRA.md) | :white_check_mark:Succeeded | |
[minecraft-marketplace/test/validation-tool.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/test/validation-tool.md) | :white_check_mark:Succeeded | |

<a name="minecraft-marketplace%2Fsubmit-content.md"></a>
### [minecraft-marketplace/submit-content.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/submit-content.md)
 - **Line 0**: **[Warning]** Invalid file link:(~/files/Required_Content_Info.xlxs).

<a name="minecraft-marketplace%2Fcreate%2Fbase-content-requirements.md"></a>
### [minecraft-marketplace/create/base-content-requirements.md](https://github.com/MicrosoftDocs/minecraft-marketplace/blob/quinn-dev/minecraft-marketplace/create/base-content-requirements.md)
 - **Line 47**: **[Warning]** Invalid file link:(~/create/urltbd).
 - **Line 56**: **[Warning]** Invalid file link:(~/create/change-log.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C28%5Cb1c7400c-ce33-0fc1-29a2-3e715f040480%5CPullRequest%5C201706280825536337-2%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mattwojo** [comment]
Hmm...I can look further in to the spreadsheet issue once I'm back on campus (probably tomorrow)... you are just trying to link out to an excel spreadsheet? Where is it hosted? (In the repo? Might need to be somewhere like Sharepoint to open properly).

On Jun 28, 2017, at 1:25 AM, Quinn Richter <notifications@github.com<mailto:notifications@github.com>> wrote:


Bummed that the link to the spreadsheet doesn't work, will have to find a different way to do that.

I kept the names Skin Pack, World Template, and Resource Pack capitalized, let me know if that's against the style guide.

Labeled the page BETA.

Added some //TODOs :)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/MicrosoftDocs/minecraft-marketplace/pull/2<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=iVSWdWR%2FDBzIvnIPbi%2BdPZwBeMmi43rWztyEIQ0GQEk%3D&reserved=0>

Commit Summary

  *   Changing folder name to submit
  *   Slack into the get-started page
  *   JIRA and how totest content
  *   Marketing Assets, how to package stubs, submission folder structure and explainations
  *   Create and Test stubs
  *   Adding the spreadsheet they need to fill out
  *   Stub of how to create content
  *   More marketing and submission
  *   More submission and content ingestion
  *   updated TOC
  *   bug fixes
  *   bug fix 2
  *   slight tweaks
  *   Let's keep this in BETA

File Changes

  *   M minecraft-marketplace/TOC.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-0&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=URGmq4%2FhXXtxVns%2F%2BpPFxPcW0JicRBWWcY7DpleP4gs%3D&reserved=0> (11)
  *   M minecraft-marketplace/create-content.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-1&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=5W6uofWJ9Tgv0JwynDmMNc5vcT54iH4AvLKy45pF9DI%3D&reserved=0> (6)
  *   R minecraft-marketplace/create/content-requirements-change-log.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-2&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=n%2FXLiOPypRiXX8kK1X9YLN%2FIBRnAw4qaO5hf7eIow8o%3D&reserved=0> (2)
  *   A minecraft-marketplace/create/package-resource-pack.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-3&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=SJtt%2BOUj8z64bL4IijrJ0jcFX135HYzllNMpbk6W2Kk%3D&reserved=0> (15)
  *   A minecraft-marketplace/create/package-skin-pack.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-4&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=EADpUhJ9MlKo8Wq6G%2FOINqoVgkZlwLtj2DN72a1WVag%3D&reserved=0> (15)
  *   A minecraft-marketplace/create/package-world-template.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-5&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=7Z3XGEwqWQ%2BfUSACJN6gquQnYajNZz4FVtbY2ic1tEA%3D&reserved=0> (15)
  *   A minecraft-marketplace/files/Required_Content_Info.xlsx<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-6&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=W513hL8vcwrI8lrSrKVmAozd9rXmhxQrEeY%2BQG%2BTi8Y%3D&reserved=0> (0)
  *   A minecraft-marketplace/files/skinTemplate.png<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-7&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=fSTL7xrmzZQGuL5LxCpPPTQoLCIBSkmOvI%2FWHtef7W8%3D&reserved=0> (0)
  *   M minecraft-marketplace/get-started.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-8&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=jeMWFqME2OmH%2B3qV8Ms4K9%2FzySQfVDw6k0xsFExdI%2Fw%3D&reserved=0> (6)
  *   M minecraft-marketplace/index.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-9&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=yYLVu693ioZX4ZcDKtr1mYsRM%2BXsB%2FruRz29b61%2FPp0%3D&reserved=0> (6)
  *   M minecraft-marketplace/submit-content.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-10&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=nqu5%2BFNoYA9av%2FvxWEkmROupmoFXPKSlxi3yYCjMJLY%3D&reserved=0> (94)
  *   R minecraft-marketplace/submit/content-ingestion-process.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-11&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=JmBltUk2Uk0EvbhMb1L1Oqc4i63gP6F4nnuqv0tlbRE%3D&reserved=0> (15)
  *   A minecraft-marketplace/submit/marketing-assets.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-12&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=4ZqKjwZMAQRqej1F20pYdhJ5wIYmwty8GukTTKqSLZU%3D&reserved=0> (37)
  *   M minecraft-marketplace/test-content.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-13&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=SKZHf6iDLv7fFrVdqiQOBppYHfeqDnQO5yAPG74yoTY%3D&reserved=0> (27)
  *   A minecraft-marketplace/test/JIRA.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-14&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=CRm986wG%2FIsHg2K8%2F3raaJsE5XaaysBIRpYWveAvoPE%3D&reserved=0> (57)
  *   M minecraft-marketplace/test/validation-tool.md<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2%2Ffiles%23diff-15&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=m%2F4yMZhcM9rFLE0ALfX8etiQLXUIKFtUjV432bXI5nw%3D&reserved=0> (1)

Patch Links:

  *   https://github.com/MicrosoftDocs/minecraft-marketplace/pull/2.patch<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2.patch&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=zNf18qZwolcvCUB568eY5KFtjGX7WQGA1lKso3LYeVc%3D&reserved=0>
  *   https://github.com/MicrosoftDocs/minecraft-marketplace/pull/2.diff<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fminecraft-marketplace%2Fpull%2F2.diff&data=02%7C01%7Cmattwoj%40microsoft.com%7C5d43f2ac5cfd49a80e9108d4bdff4a91%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636342351544884496&sdata=5phQUlqot7LcXhCNVcdBRtJkV2IoZvMY38LOucrQbZk%3D&reserved=0>

File: minecraft-marketplace/get-started.md [hunk 1]
```diff
@@ -12,4 +12,8 @@ keywords: store, partner
 
 # Get started with Minecraft Marketplace
 
-Over 100 million people play Minecraft today and now with the Minecraft Partner Program, you can be a part of creating new worlds and gameplay for this incredible audience through the in-game Store of maps, skins, and textures.
+Congratulations on being selected to join the Minecraft Partner Program! We're super excited for you to start building and selling content on the Marketplace.
```
Comment (shawjohn) [inline minecraft-marketplace/get-started.md]: Note that people might get to this page who have not been selected to join the Minecraft Partner Program.

**Comment by shawjohn** [comment]
Quinn, I think the capitalization is fine for the items you mentioned.

**Pull request merged**